wunderio/frontend-project
# PSW-27: Upgrade Husky and Lint-staged to latest versions.

Link to ticket: https://wunder.atlassian.net/browse/PSW-27

Changes proposed in this PR:
- Updated Husky and Lint-Staged to latest versions.
  - There are changed a way how to configure lint-staged and execute husky. 
  - Updated has been done by official guidelines: https://typicode.github.io/husky/#/?id=migrate-from-v4-to-v6
- Added package name, which is required for `npm ci` command.

How to test:
1. `npm install`
2. Make some error in JS file so eslint catches that as an error.
3. Try to commit. Should not be able to commit if there is an eslint error.

Link to feature environment:
https://feature-psw-27-husky.frontend-project.dev.wdr.io/
